components/chef-ui-library/src/charts/chef-radial-chart/chef-radial-chart.tsx

```diff
@@ -86,6 +86,11 @@ export class ChefRadialChart {
    */
   @Prop() id: string;
 
+  /**
+   * The width of the gap to apply between chart segments.
```

Comment:
This comment should mention units:
```suggestion
   * The width of the gap (in radians) to apply between chart segments.
```